l10n_ch_qr_bill_without_amount/models/account_move.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2024 Camptocamp (<https://www.camptocamp.com>).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    qr_bill_amount = fields.Char(
+        compute="_compute_qr_bill_amount",
+    )
+
+    @api.depends("amount_residual")
```

Comment:
```suggestion
    @api.depends("amount_residual", "payment_mode_id.qr_bill_without_amount")
```

Reply:
Thank you @TDu , what a pity I didn't find the model in version 17 before. It makes more sense to backport this module and create a glue module.

Reply:
@TDu I just realized that the 17.0 module is not working as expected.
When we scan th QR we still see the amount:
![image](https://github.com/user-attachments/assets/c5f8ca54-f7a5-4f37-8a4c-422354febfea)
You can try it here : https://qr-rechnung.net/#/scan

Reply:
Another thing that is not possible is to print invoice **with** and **without** amount in batch.

Reply:
Ok, can we keep in the readme or roadmap of this module that the other one exists and we should try to merge them or regroup their common functionality in some way...
Also I think there still is a missing field in the api.depends (see comment above)

Reply:
> When we scan th QR we still see the amount

That's a bug of the module...

> Another thing that is not possible is to print invoice with and without amount in batch

Feel free to provide an improvement on the existing module, there's no reason to have two different modules for that